brazilian-journal-of-veterinary-research-and-animal-science.csl

```diff
@@ -0,0 +1,332 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
+  <info>
+    <title>Brazilian Journal of Veterinary Research and Animal Science</title>
+    <title-short>BJVRAS</title-short>
+    <id>http://www.zotero.org/styles/brazilian-journal-of-veterinary-research-and-animal-science</id>
+    <link href="http://www.zotero.org/styles/brazilian-journal-of-veterinary-research-and-animal-science" rel="self"/>
+    <link href="http://www.zotero.org/styles/vancouver" rel="template"/>
+    <link href="https://www.revistas.usp.br/bjvras/about/submissions" rel="documentation"/>
+    <author>
+      <name>Alberto Kawai</name>
+      <email>alberto.kawai@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>1413-9596</issn>
+    <eissn>1678-4456</eissn>
+    <updated>2020-02-03T10:52:38+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="pt-BR">
```

Comment:
@amkawai, since this journal appears to only publish in English, it would be better to fix the style to US or British English (`default-locale="en-US"` or `"en-GB"` on `<style>`), then to override the Portuguese translations with English ones here.

Could you make this change?

Reply:
Thanks! Done.